getting-started/chat-thread.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Chat and thread"
+weight: 10
+---
+
+# Chat and thread
+
+
+### Basics
+
+In our system chat is divided into threads. Threads are standalone parts of chat and can contain events. In every chat multiple users (customers or agents) can participate and every user can have multiple chats at the same time.
+
+New threads within chat are created on server side, algorithm of this is discussed below in *Advanced* section.
+
+![Chats and Threads](../images/chats.png "chats and threads")
+
+Events are some portion of data, that can be send to chat (in eg. messages), they are discussed in the next section of this guide.
```

Comment:
Events are some portion of data, that can be send to chat (in eg. messages), they are discussed in the next section of this guide. -> Events are portions of data which can be sent to a chat (e.g. in messages). See the next section of this guide for details.